src/main/java/com/jwplayer/southpaw/Southpaw.java

```diff
@@ -715,6 +715,7 @@ protected String createParentIndexName(Relation root, Relation parent, Relation
     public static void deleteBackups(Map<String, Object> config) {
         BaseState state =  new RocksDBState(config);
         state.deleteBackups();
+        state.close();
```

Comment:
do we want to wrap this effect within a try/catch?

Reply:
Wrapped in a try/finally. `close()` does not appear to throw